### DISCUSSION TITLE
how to unbind some key bindings?

### DISCUSSION THREAD

**Discussion by funkiestj** [whitphx/vscode-emacs-mcx#1379] [Q&A]
I'm not even sure what key binding I'm hitting because it is muscle memory but I keep closing my VSCode session by accident.

In regular emacs, I have C-x C-c unbound so that I don't accidentally close emacs when I don't mean to.

Is there way to turn off all the "close <thingie>" key bindings?  I'd really prefer to use the mouse and menu to close things.  If not, I'd like the ability to disable all "close <thingie>" keybinds as an option in settings .

----
background: I've used emacs for decades (and still use it).  I'm new to VSCode.  I'm not sure what key sequence I'm hitting that closes my current window (and perhaps more) but I think it involves the right hand, e.g. C-k.  I'm pretty sure I'm NOT hitting C-x C-c but I'd prefer to unbind it regardless.

P.S. thanks for the extension -- it really helps make VSCode more usable for me, even with the annoyance described above.

**Comment by whitphx** [comment]
https://code.visualstudio.com/docs/getstarted/keybindings#_removing-a-specific-key-binding-rule
